build/install.sh

```diff
@@ -1,20 +1,22 @@
 #!/bin/bash
+set -eu -o pipefail
+
 echo "==========STARTING INSTALL========="
 echo "Installing unzip............"
-apt -y install unzip
+hash unzip 2> /dev/null || apt -y install unzip
 echo "Installing packer..."
 echo "$PWD"
-curl -L "$PACKER_ZIP" -o packer.zip && unzip packer.zip
-./packer version
-
-# Check if $SPEL_ACCESS_KEY is not empty
-if [ -n "$SPEL_ACCESS_KEY" ]; then
-    export AWS_ACCESS_KEY_ID=$SPEL_ACCESS_KEY
-    export AWS_SECRET_ACCESS_KEY=$SPEL_SECRET_KEY
+if ! hash packer 2> /dev/null
+then
+  curl -L "$PACKER_ZIP" -o packer.zip
+  unzip packer.zip
+  mkdir -p "${HOME}/.local/bin"
+  mv packer "${HOME}/.local/bin"
 fi
+packer version
 
 # Check if $SPEL_SSM_ACCESS_KEY is not empty
-if [ -n "$SPEL_SSM_ACCESS_KEY" ]; then
+if [ -n "${SPEL_SSM_ACCESS_KEY:-}" ]; then
```

Comment:
If we're going to change this, do we want to move towards `[[ <STUFF> ]]` notation, instead (to be more-broadly consistent across the various BASH-based content?

Same question for remaining `if [ <STUFF> ]` content

Reply:
Oh sure, that wasn't on my radar, but easy peasy